lab_5_scrapper/scrapper.py

```diff
@@ -1,82 +1,205 @@
 """
 Crawler implementation
 """
-from typing import Pattern, Union
+import datetime
+import json
+import locale
+import random
+import re
+import shutil
+import time
+from pathlib import Path
+from typing import List, Dict, Pattern, Union
+from urllib.parse import urlparse
+
+import requests
+from bs4 import BeautifulSoup
+
+from core_utils.article.article import Article
+from core_utils.article.io import to_raw, to_meta
+from core_utils.config_dto import ConfigDTO
+from core_utils.constants import ASSETS_PATH, \
```

Comment:
strange indents for multiline imports. usually when they are multiline, they are surrounded with brackets